File: APEC.RRHH.api/Core/Models/CandidateInterview.cs
using System;
using Core.Contracts;

namespace Core.Models
{
    public sealed class CandidateInterview : IEntityBase
    {
        public Guid Id { get; set; }
        public Guid CandidateEmployeeId { get; set; }
        public CandidateEmployee CandidateEmployee { get; set; }
        public string EmployeeNote { get; set; }
        public Guid CandidateEmployeeAspiratedJobId { get; set; }
        public CandidateEmployeeAspiratedJob CandidateEmployeeAspiratedJob { get; set; }
        public Guid JobId { get; set; }
        public Job Job { get; set; }
        public DateTime InterviewDate { get; set; }
        public Guid EmployeeId { get; set; }
        public Employee Employee { get; set; }
        public bool Hired { get; set; }
    }
}

File: APEC.RRHH.webapp/src/app/models/candidate-interview.ts
import { CandidateEmployee } from './candidate-employee';
import { CandidateEmployeeAspiratedJob } from './candidate-employee-aspirated-job';
import { Employee } from './employee';
import { Job } from './job';

export class CandidateInterview {
    id: string;
    candidateEmployeeId: string;
    candidateEmployeeAspiratedJobId: string;
    candidateEmployee: CandidateEmployee;
    candidateEmployeeAspiratedJob: CandidateEmployeeAspiratedJob;
    jobId: string;
    job: Job;
    interviewDate: Date;
    employeeIdWhoInterviewed: string;
    employee: Employee;
    employeeNotes: string;
    hired: boolean;
}
